index.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs')
+const p = require('path')
+const maps = {}
+const folder = p.join(__dirname,'maps')
+
+var files = fs.readdirSync(folder).map(f => {
+  try {
+      maps[p.basename(f, p.extname(f))] = require(p.join(folder,f))
```

Comment:
@phanisaripalli here we load all the json files in maps folder to a key in the module's object.
on the other projects we can do
```
const eyzmap = require('eyzmaps')
console.log( eyzmap['iso639-2-language'].es)
```